Swat/SwatAccordion.php

```diff
@@ -93,13 +93,20 @@ public function display()
 
 			// toggle link
 			$title = ($page->title === null) ? '' : $page->title;
-			$wrapper_tag = new SwatHtmlTag('span');
-			$wrapper_tag->class = 'swat-accordion-page-toggle';
+			$wrapper_span = new SwatHtmlTag('span');
+			$wrapper_span->class = 'swat-accordion-page-toggle';
+			$wrapper_span->open();
+
+			$anchor_tag = new SwatHtmlTag('a');
+			$anchor_tag->href = '#'.$page->id;
```

Comment:
The url for this seems to be wrong? It should be relative to the page, but it redirects to the main boardreview sales page instead? Not sure why this would be wrong, when I'm using the original code? @m-mitchell Can you look into this too please?

Reply:
I think I figured it out, I'll merge this patch and fix it in my PR. We are using the base tag to set the base URL to the homepage: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base

...so in order to get hashtag links to work properly, we have to manually add the real base URL to it, which is done here (and needs to be updated for the new code): https://github.com/silverorange/swat/blob/master/www/javascript/swat-accordion.js#L87